sdk/docs/core/az_mqtt_rpc.md

```diff
@@ -2,7 +2,12 @@
 
 ## RPC Client HFSM
 
-![mqtt_rpc_client](https://www.plantuml.com/plantuml/png/dLPRSzf657xdLvotI1qClRnv4ayeLbgDOo2Ix9cqT3GIMl0sOfVkhaBSZFzxpzaL8B00kqzeRz_vpdTklDScKwPQbfqtm1T2AWRcWMa6cOQ7J1GbAq1MHlE2WPpZ6TTGy6oXikKbt7vAaZIu3qP9EXoFHi4mIQFWKxTrFvxixL-7yR4dpQDuWhm6MbVBbU5Iw4jS_jY3E_55oBK0zfK9eo53rdEfrfagaAx71Vd05KZ55rna9J9MR4O0NM0SpnHmaSjAe6-AtBFVVW_1V-f1E8TeCao77qDY4mM3wyzfEBf6ERHuOo0h5SkAHyWP4w3HVhTvFRxxbHm2b6bHPIeJXY4b8q4nlP9eS8tsDoJQp7I9qkyVujjQZI2AnX5mafw1hd8zKpnd2FTFnRG1mvSkGaJ-vruxypKlIn3I86VB69Okpfa0AMOCAb4ohQs5MSdfGYMsfZGHMZ2KkR5PaBbVUeHMfBMMNRVQo6cZKcddFPsWvXYaPKPne3qobwrMW5a6WfcrL5-iIXIfhwnyx7Kwc9o6GRIQ3Hsjxnq0jnTYPkA2Vk0ItkPN3SV3qVtu9g0mzD_syoi4DPKIdXKLpmOVm-iqJWP9u1z3k1lLSNG03SSzlxFby26hIiCANDTV3WU9gaVti2cFTnXK__3XOFGPxFaBQ0DiQSGrDouMkDTOFQT3jMvJHhNU_CRDvg9S7RztNEgfc9gJj4zXxKbUemp-ICStFlcwTJUorVVTQj3OIICfZTt4HCVwu8i7Gnthu10WeHRc7k87LONh5p2J1Ul2tnLL5Iwn9suw7JKNh1rZi6Qf-BESbznWWZh4qE3kd2bDMPqpygs6T5ojiIxabR7c9bLUSls0rnm0bGPnRdYqks8tTyip8MsvsXxJvnVFrm_Tc62Cmb4wYSQ_HK4SssA0XiPmSVKAvAdOwnYU7G8JbmR-8SiBX6uOij2DgO7F1iCRcXWUHdqc5NLyu8LVMy8EwiqpNbQA-VjrsU1dwv8m_Ab0U9l3INWRZE-I9k6JBv7tKwVMfC7IkJMyfuRyl3caV9vYVFWinVuumm7i-ViOP-galTeEEHmgvI3SsAT8_ld3NnJ0PcybrLzGj0JKXvPenhjeRl7EIZCL3ih8rCxQTdMQj0XshvSsGqUGBVVRrObFvj7HpDaEMvSvfq9-sYl7wg8Em-icqtCV_yTqU-hivkrEoiRSUj5eiqqubyR8fUt26qZasE_3mFuPWtabx8UcvdvQ6hlgkZzyTXwu0K1LRO_WtJlxtoSPJyAXNvy26Fm7fZdPI5UCAS-gwzU7vsDPNqopVzE7kLwczeNbu3acziAJBVaEdAbekte5zwFqN9DfsDsBuCv0Ipcw66qJmE4SYaH9uzhOTZIznoySYFy1 "mqtt_rpc_client")
+![mqtt_rpc_client](https://www.plantuml.com/plantuml/png/rLPDR-Cs4BthLymjKk5r-d4BTeDr7GKrajXUMmwmRGg1aaOE4ObKIIffMkn_BuUILSkhk3xqieSW8cVuvisyuT0VjM7Ar6NXdG7V2gaGp2Dg1ARXaOcim0naRJJF46HkRLn3njbMiV82xZv5KHpUX_CedYxcyt0QnQlmqsXmVxtixL_DrlqZGoOwPNbDjAxBod0fz8LTVZ-6ZNWIyaK0FgCm6fX2oALwOIh337mkA0UkG2g-vO8Lbh72b016WDpQ572Ho5fareNISz_1s8B_C8PP3glbD9xUpeZDAfnSVOvdyojR3YtE3B128SjU8K4Ke6tyqRxxUlCp9GHSmxPcYWc3c86He51NqWPyiV5R4eDXHaJfb-lrNLkDSBLQh81JwH5qdUXKyGGjt1yrQWE6bur2HFx7yI2aZLaK8AInd1rZA1kTcG0fKeHQ5AYrYv0Md1ngqONIH6YBHkzlPXJkfp6mgefJLXG9Ivz6X9t5RMNRrQwuJgDQDxKW2t1X9IiPgK9x5DoYZRhCAbdmz1LIMHGxxPWz9j2yIFNawadxprYyZZrF6sOGLbKwRHBusmDezcPPWHwuVt01xvFBlHBDvlUBcv04Etz_dbnQK5ChuLFvoJQvdLt5wsWIXS5R29ivATu1z7BskwRUymkyZjargmk3cGUxhmOTXL6l3gvNNl-mAOB3WptjxnRt_oRs81xClZZ-e6im8Z-5fNp6Gnbo9Si1IUX0nRJUcKfxMBq2yGKZYR_TxZHEqE8WddsLycJ-6Lnc9yW0q1LmtROZ5riFzbU7XIH_rLmMDUZv9pSxHraD-nrl7Ih4KTh7i7ejrA9CVeqNDm5Gj_pstU_mxhBNoE_i-WH40ZmVQigk6VVmLZJM6YJditG-qtiQ8MsE3PIHiIBVcEVsk-9fN3AJFgBs3nCbs9xGhZRHx2vSR2A2xFiFqWZlsuGcqnjw47oPGoeL3NJWMJ1mKOPcOF0WtiphbAmSA-9toXXu4AVCWLxg_o-q1qVLXFqQkbo3hebv3dx5b2PEkavJn4mtW-5WXDX0ct1DB_pTyZQqJTxulAd4UkayiD1u1ErwihcDmglZO3J_2B0JvXkSVyEyIS7JNgFk_B6fnB8iJcLPCf4vgWEzGttHvJuW2M7E8h-g4xfj57duWYwx2-eYzv06tNfG_SdmDSA-DV2-UDr3tntkFOqdlPv26eH46YDBa3bqa1x0-JbCdC2Gry9zQ0_-QqourQZvJHg01-ut203FeJ71Xmz0KaMBvMmQj5O0evZGd6B45QBoNKs2ndZ2N6iTO_PN_9ZhCdOa7GU_OU8SldZqPqr0MNiVxLTT5ly0 "mqtt_rpc_client")
```

Comment:
Don't forget to update from your other PR https://github.com/Azure/azure-sdk-for-c/pull/2763

Reply:
Same for the one below

Reply:
done!